GitUI/CommandsDialogs/FormCleanupRepository.resx

```diff
@@ -1,61 +1,61 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
+  <!--
     Microsoft ResX Schema 
-    
+
     Version 2.0
-    
-    The primary goals of this format is to allow a simple XML format 
-    that is mostly human readable. The generation and parsing of the 
-    various data types are done through the TypeConverter classes 
+
+    The primary goals of this format is to allow a simple XML format
+    that is mostly human readable. The generation and parsing of the
+    various data types are done through the TypeConverter classes
     associated with the data types.
-    
+
     Example:
-    
+
     ... ado.net/XML headers & schema ...
     <resheader name="resmimetype">text/microsoft-resx</resheader>
     <resheader name="version">2.0</resheader>
     <resheader name="reader">System.Resources.ResXResourceReader, System.Windows.Forms, ...</resheader>
     <resheader name="writer">System.Resources.ResXResourceWriter, System.Windows.Forms, ...</resheader>
     <data name="Name1"><value>this is my long string</value><comment>this is a comment</comment></data>
-    <data name="Color1" type="System.Drawing.Color, System.Drawing">Blue</data>
+    <data name="Color1" type="System.Drawing.Color, System.Drawing"">Blue</data>
```

Comment:
Why has this changed?

Reply:
I'm not sure, this was done automatically. Should I revert the file?

Reply:
This doesn't look like it's automatic, please revert. Though, if you can positively reproduce it - please file a bug via the VS Feedback button in VS (top right corner).

Reply:
I'll look into it. I definitely didn't mess with the .resx files manually, at least that I am aware of. Visual Studio doesn't make it straightforward to modify at that level.